plugin.go

```diff
@@ -200,6 +200,11 @@ func commandBuild(build Build) *exec.Cmd {
 	if build.Compress {
 		args = append(args, "--compress")
 	}
+    if build.Cache {
+        args = append(args, "--pull=false")
+    } else {
+        args = append(args, "--pull=true")
+    }
```

Comment:
also make sure to run `gofmt` on your changes. This will ensure the codebase conforms to the official Go style guidelines (in this case tags for indentation)